index.bs

```diff
@@ -1200,6 +1205,7 @@ The "Version(s)" column in the following table lists the version(s) of the boxes
     - <a href="https://github.com/AOMediaCodec/av1-avif/pull/240">Adopt MIAF shared constraints</a>
     - EDITORIAL: <a href="https://github.com/AOMediaCodec/av1-avif/pull/251">Clean up usage of dfn and linking</a>
     - <a href="https://github.com/AOMediaCodec/av1-avif/pull/250">Clarify required versions of non-essential item properties</a>
+    - EDITORIAL: <a href="https://github.com/AOMediaCodec/av1-avif/pull/257">Formatting changes in clause 1 and 2</a>
```

Comment:
This pull request is more than just formatting changes. For example, it changes four instances of `AV1CodecConfigurationBox` to `AV1ItemConfigurationProperty`.

Reply:
> The syntax and semantics of the <dfn export>`AV1ItemConfigurationProperty`</dfn> are identical to those of the <code>[=AV1CodecConfigurationBox=]</code> defined in [[!AV1-ISOBMFF]], with the following constraints

To me the change is a no-op spec-wise, because this is an equivalent box (`AV1CodecConfigurationBox` -> `AV1ItemConfigurationProperty`).  
But the constraints rather apply to the latter than to the former, because `AV1ItemConfigurationProperty` is used in AVIF, not really `AV1CodecConfigurationBox`.

Happy to reword the PR title though.

Reply:
Thanks. I was just pointing out that the old PR title was not accurate. The new PR title "Add refs, fix wording and format in clauses 1,2" looks good.